documentation.html

```diff
@@ -352,6 +357,55 @@ <h4 class="display-inlineBlock">Icon Positioned to The Right</h4>
             </div>
           </section>
         </section>
+        <!-- Toggle button class starts here -->
+        <section class="instruction-block" id="toggle">
+          <h2 id="base-icon-btn">Toggle Button</h2>
+          <div class="text">
+            The <code>toggle-btn</code> class along with <code>toggle-off-btn</code> / <code>toggle-on-btn</code> state class enables you to have a toggle button.Apart from its basic use, it can be used for checkboxes and radiobuttons.
+          </div>
+          <br />
+          <section class="instruction-block" id="toggleCheckbox">
+            <h4 class="display-inlineBlock">Toggle Button as Checkboxes</h4>
+            <div class="text">
+              Adding the <code>toggle-btn toggle-off-btn</code> class to your checkbox inputs converts them to toggle buttons.For example:
+              <br><br>
+              <h5>Which of the following is/are an OS?</h5><br>
+              <input type="checkbox" id="1" class="sbtn toggle-btn toggle-off-btn" />
```

Comment:
The example should just be one checkbox with the code block for it. No need to include a full example (same thing for radio buttons) Also don't include a label as it is not necessary